public/js/render.js

```diff
@@ -0,0 +1,50 @@
+var renderModule = (function () {
+  console.log(renderModule);
+  return {
+    renderEvents: function (error, eventsArray) {
+      var listPage = document.getElementById('list-page-content');
+      var eventsSection = document.getElementById('events-section');
+      while (eventsSection.firstChild) {
+        eventsSection.removeChild(eventsSection.firstChild);
+      }
+      if (eventsArray.length === 0) {
+        var noEvents = document.createElement('h1');
+        noEvents.innerHTML = 'No Events Where Found On This Day';
+        return eventsSection.appendChild(noEvents);
+      }
+      if (error) {
+        eventsSection.innerHTML = error;
+      }
+
+      var filterEvents = eventsArray.filter(function (event) {
```

Comment:
the filter function here can be simplified.

Reply:
how?